lib/potential.cpp

```diff
@@ -63,28 +56,23 @@ double CoulombSpherePotential::V(double r)
  * @retval
  */
 CoulombFermi2Potential::CoulombFermi2Potential(double Z, double R, double A,
-                                               double thickness, int csteps)
-    : CoulombSpherePotential(Z, R)
-{
+    double thickness, int csteps)
+  : CoulombSpherePotential(Z, R) {
```

Comment:
Not sure if it's compliant with the syntax/formatting, but now the indentation's decreased could this go on one line, e.g. 
```suggestion
    double thickness, int csteps) : CoulombSpherePotential(Z, R) {
```